packaging/scripts/lib/assert-cisco-signature.sh

```
@@ -0,0 +1,148 @@
+    while IFS= read -r cn; do
+        [ -z "${cn}" ] && continue
+        if [ "${cn}" = "${_DEFENSECLAW_CISCO_PUBLISHER_CN}" ]; then
+            found=1
+            break
+        fi
+    done <<EOF
+${subject_cns}
+EOF
+
+    if [ "${found}" -eq 0 ]; then
+        die 4 "assert-cisco-signature: ${path} does not carry a certificate with CN '${_DEFENSECLAW_CISCO_PUBLISHER_CN}' (found: $(printf '%s ' ${subject_cns}))"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)